### PR TITLE
Use populate_db script

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/157

Depends on https://github.com/PerfectFit-project/virtual-coach-db/pull/9 (please first review that one)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ with `build: https://github.com/PerfectFit-project/virtual-coach-db.git#feature-
 NB: Don't commit changes to `docker-compose.yml`
 
 ### Database
-The database automatically loads the test data in `init_db_scripts/test.sql`.
-So if you want to test with different data or a modified schema you have to update this.
+The database automatically loads the test data in
```
Comment (raar1) [inline README.md:70]:
```suggestion
Test data is automatically loaded into the database using a script in 
```

File: README.md [hunk 2]
```diff
@@ -67,6 +67,13 @@ with `build: https://github.com/PerfectFit-project/virtual-coach-db.git#feature-
 NB: Don't commit changes to `docker-compose.yml`
 
 ### Database
-The database automatically loads the test data in `init_db_scripts/test.sql`.
-So if you want to test with different data or a modified schema you have to update this.
+The database automatically loads the test data in 
+`virtual-coach-db` [helper/populate_db.py](https://github.com/PerfectFit-project/virtual-coach-db/blob/ca6b5c79064c3b1dcf34ce83d1f59773f33427e3/helper/populate_db.py).
+So if you want to test with different data you have to update this.
```
Comment (raar1) [inline README.md:72]:
```suggestion
So if you want to test with different data (particularly if you have updated the db schema by changing `models.py`) you have to update this.
```

File: docker-compose.yml [hunk 3]
```diff
@@ -7,21 +7,14 @@ services:
       POSTGRES_DB: perfectfit
       POSTGRES_USER: root
       POSTGRES_PASSWORD: root
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready -d perfectfit"]
-      interval: 5m
-      timeout: 5m
-      retries: 5
```
[the remark this answers is not in the document]
Comment (raar1) [reply docker-compose.yml:14]: This explains a lot...

**Review by raar1 — APPROVED** [review]
OK looks good. I only made some minor suggestions to updates for the README. We will come back to the ridiculous docker-compose depends_on filth at a later date :)

File: docker-compose.yml [hunk 4]
```diff
@@ -7,21 +7,14 @@ services:
...
     ports:
       - "5432:5432"
 
   manage_db:
-    build: https://github.com/PerfectFit-project/virtual-coach-db.git#main
+    build: https://github.com/PerfectFit-project/virtual-coach-db.git#use-population-script-in-main
     tty: true
-    depends_on:
-      - db
+    restart: on-failure
```
[the remark this answers is not in the document]
Comment (svenvanderburg) [reply docker-compose.yml:16]: Yep. There is no immediate cleaner way for it it seems, at least not what I got from https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y and:
image

**Pull request merged**